app-text/sigil/sigil-0.9.16.ebuild

```diff
@@ -4,6 +4,7 @@
 EAPI=7
 
 PYTHON_COMPAT=( python3_{5,6,7} )
+PYTHON_REQ_USE="tk"
```

Comment:
how does this change relate to `USE=plugins`? shouldn't it be in the version bump?

Reply:
I put it in the wrong commit as I found about this missing when changing `USE=plugins`.
Fixed.